src/deprecated.jl

```diff
@@ -0,0 +1,116 @@
+"""
+    qubits(N::Int; mixed::Bool=false)
+    
+    qubits(sites::Vector{<:Index}; mixed::Bool=false)
+
+
+Initialize qubits to:
+- An MPS wavefunction `|ψ⟩` if `mixed = false`
+- An MPO density matrix `ρ` if `mixed = true`
+"""
+qubits(N::Int; mixed::Bool = false) =
+  qubits(siteinds("Qubit", N); mixed = mixed)
+
+function qubits(sites::Vector{<:Index}; mixed::Bool = false)
+  @warn "Method `qubits` is deprecated, use `trivialstate` or `trivialprocess` instead."
+  ψ = productMPS(sites, "0")
+  mixed && return MPO(ψ)
+  return ψ
+end
+
+"""
+    qubits(M::Union{MPS,MPO,LPDO}; mixed::Bool=false)
+
+Initialize qubits on the Hilbert space of a reference state,
+given as `MPS`, `MPO` or `LPDO`.
+"""
+qubits(M::Union{MPS,MPO,LPDO}; mixed::Bool = false) =
+  qubits(hilbertspace(M); mixed = mixed)
+
+
+"""
+    qubits(N::Int, states::Vector{String}; mixed::Bool=false)
+
+    qubits(sites::Vector{<:Index}, states::Vector{String};mixed::Bool = false)
+
+Initialize the qubits to a given single-qubit product state.
+"""
+qubits(N::Int, states::Vector{String}; mixed::Bool = false) =
+  qubits(siteinds("Qubit", N), states; mixed = mixed)
+
+function qubits(sites::Vector{<:Index}, states::Vector{String};
+                mixed::Bool = false)
+  @warn "Method `qubits` is deprecated, use `trivialstate` or `trivialprocess` instead."
+  N = length(sites)
```

Comment:
Instead of having all of this code around, can't we just call `strivialstate`?

Reply:
Yes of couse